spec/features/admin/budget_investments_spec.rb

```diff
@@ -508,6 +508,33 @@
       expect(page).not_to have_content "Refugees, Solidarity"
     end
 
+    scenario "Shows alert when checks valuation as finished", :js do
+      budget_investment = create(:budget_investment)
+
+      visit admin_budget_budget_investment_path(budget_investment.budget, budget_investment)
+      click_link 'Edit dossier'
+
+      expect(page).to have_content 'Valuation finished'
+
+      find_field('budget_investment[valuation_finished]').set(true)
+
+      page.accept_confirm("Are you sure you want to mark this report as completed? If you do it, it can no longer be modified.")
```

Comment:
I wonder if both scenarios should finish just by accepting the alert... or by checking that after accepting the alert something expected happens. Also ... there are no other specs that where checking the "Finish valuation" scenarios? and if so... wouldn't those be affected by this change? 🤔